packages/next/src/executors/build/build.impl.ts

```diff
@@ -45,7 +45,19 @@ export default async function buildExecutor(
     (process.env as any).__NEXT_REACT_ROOT ||= 'true';
   }
 
-  await build(root);
+  // Get the installed Next.js version (will be removed after Nx 16 and Next.js update)
+  const nextVersion =
+    readJsonFile('node_modules/next/package.json')?.version ?? '';
```

Comment:
you can do `require('next/package.json')` which should always have a valid export point.

`node_modules` is not guaranteed if using yarn/pnpm pnp for example.